segbot_arm_tutorials/src/ex1_subscribing_to_topics.cpp

```diff
@@ -0,0 +1,144 @@
+#include <ros/ros.h>
+#include <ros/package.h>
+
+#include <signal.h>
+
+#include <sensor_msgs/JointState.h>
+
+#include <geometry_msgs/PoseStamped.h>
+
+#include <jaco_msgs/FingerPosition.h>
+
+#define NUM_JOINTS 8 //6+2 for the arm
+
+//global variables for storing sensory data
+sensor_msgs::JointState current_state;
+geometry_msgs::PoseStamped current_pose;
+sensor_msgs::JointState current_efforts;
+jaco_msgs::FingerPosition current_finger;
+
+
+bool heardJoinstState;
+bool heardPose;
+bool heardEfforts;
+bool heardFingers;
+
+//true if Ctrl-C is pressed
+bool g_caught_sigint=false;
+
+/* what happens when ctr-c is pressed */
+void sig_handler(int sig) {
+  g_caught_sigint = true;
```

Comment:
Can you ensure that the spacing in this method (and several of the following ones) match the spacing of the rest of the document? i.e. 2 spaces vs 4 spaces